configure.in

```diff
@@ -4,7 +4,35 @@ AC_CONFIG_AUX_DIR([./buildutils])
 AC_CONFIG_MACRO_DIR([./buildutils])
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([include/exc_platform.h])
-AC_CONFIG_HEADER([include/config.h Driver/atool/src/include/nata/nata_config.h])
+AC_CONFIG_HEADER([include/config.h Driver/atool/src/include/nata/nata_config.h ])
+
+#--------------------------------------------------------------------
+# Bash function for trimming strings
+trim ()
+{
+    local var="$*"
+    # remove leading whitespace characters
+    var="${var#"${var%%[![:space:]]*}"}"
+    # remove trailing whitespace characters
+    var="${var%"${var##*[![:space:]]}"}"
+    printf '%s' "$var"
+}
+#--------------------------------------------------------------------
+
+AC_ARG_WITH(version-tag,
+	    [  --with-version-tag=<tag> Add arbitrary string tag to the xcodeml-tools version],
+		[with_version_tag_specified=yes], [with_version_tag_specified=no])
+
+if test "x${with_version_tag_specified}" = "xyes"; then
+  # Trim tag string
+  with_version_tag=$(trim "${with_version_tag}")
```

Comment:
This won't work otherwise?

Reply:
It will (technically) work without "trim", but I didn't want leading & trailing spaces to be part of the tag.